### DISCUSSION TITLE
请问下，VcPostProcessStageScan 的例子是在3d地图下运行的，我调成2d地图就失效了，请问该怎么改？

### DISCUSSION THREAD

**Discussion by KKindom** [zouyaoji/vue-cesium#234] [Q&A]
希望大佬回复下谢谢了

**Comment by zouyaoji** [comment]
你有测试原生的 PostProcessStage 支持 2d 模式不？

**Reply by HHANG52** [reply]
我想问一下，引入这个，缩放地图的时候， VcPostProcessStageScan在地图上来回抖动，是怎么解决。

**Reply by zouyaoji** [reply]
开一下深度检测会好一点

**Reply by HHANG52** [reply]
viewer.scene.globe.depthTestAgainstTerrain = true
是这个吗，我在 <vc-viewer @ready="onViewerReady"></vc-viewer>    的onViewerReady里写了，就像官网示例一样，还是存在地图放大缩小的时候抖动的情况。

**Reply by zouyaoji** [reply]
别用后处理了，效果不好，用材质。有个材质叫“VcCircleWave”。
参考
https://zouyaoji.top/vue-cesium/#/zh-CN/component/datasources/vc-datasource-custom

**Reply by HHANG52** [reply]
好的 ，感谢。
